### DISCUSSION TITLE
Retry mechanism

### DISCUSSION THREAD

**Discussion by Sushanti99** [milvus-io/milvus#4835] [Q&A and General discussion]
milvus = Milvus(host='', port='')

Is there someway to specify `max_retry` here?

**Comment by yhmo** [comment]
Try this:

`MILVUS = Milvus(host='', port='', max_retry=5)`

**Reply by Sushanti99** [reply]
I'm not sure what the right terminology is.. 

`MILVUS = Milvus(host='', port='', max_retry=5)` works
`MILVUS = Milvus(host='', port='', retry=5)` works too

So is the arg passed being considered?

**Reply by yhmo** [reply]
The max_retry default value = 3. 
`MILVUS = Milvus(host='', port='', max_retry=5)`   The internal logic will retry 5 times.
`MILVUS = Milvus(host='', port='', retry=5)`    The "retry=5" is invalid, the internal logic will retry 3 times(default value).
